.github/workflows/push-notification-jar.yml

```diff
@@ -0,0 +1,47 @@
+name: Upload Notification Jar to Maven
+
+on:
+  push:
+    tags: 
+      - v*
+jobs:
+  upload-notification-jar:
+    runs-on: [ubuntu-16.04]
+    strategy:
+      matrix:
+        java: [12]
+    name: Upload Notification Jar to Maven
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v1
+        
+      - name: Configure AWS CLI
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
```

Comment:
I don't think we are ever going to parallelize runs across different java versions for a specific version of the plugin. Can we remove the java matrix and just set the java-version here? Ideally we could get the java version from some property file so we don't have to update this though.

Reply:
I was under the impression that JAVA_HOME path needs to point to Java version 12. 
If that is not the case then I can go ahead and remove it.

Reply:
removed java matrix as per our call